centre-web/src/components/AuthManagement/EditAccess/index.tsx

```diff
@@ -47,15 +47,29 @@ export const EditAccessModal = ({
   });
 
   const { setClose } = useModal();
-  const [selectedRole, setSelectedRole] = useState<string | null>(
-    app.group_path ?? null,
-  );
-
+  
   const { data: accessLevels = [], isLoading: accessLevelsLoading } =
     useGeteApplicationAccessLevels({
       appName: app.name,
     });
 
+  const [selectedRole, setSelectedRole] = useState<string | null>(null);
+  useEffect(() => {
+    if (accessLevelsLoading || accessLevels.length === 0 || !app.group_path) {
+      return;
+    }
+    const normalizePath = (path: string) => path.replace(/^\/+/, '');
+    const normalizedAppPath = normalizePath(app.group_path);
```

Comment:
Could we simplify the code a little bit?

could we perhaps just check level.group_path.includes(app.group_path)